lambda/github_api.py

```diff
@@ -0,0 +1,97 @@
+import os
+import time
+
+import requests
+
+import config
+from logger import LOG
+
+TOKEN_PREFIX = "/alert-processor/tokens/github/"
+API_ROOT = "https://api.github.com"
+ACCESS_TOKEN = None
+
+
+def set_github_access_token(token):
+    """ Get the access token once """
+    global ACCESS_TOKEN
+    ACCESS_TOKEN = token
+
+
+def get_github_access_token():
+    """ Get the access token once """
+    global ACCESS_TOKEN
+    if ACCESS_TOKEN is None:
+        access_token_param = os.environ.get("GITHUB_TOKEN")
+        ACCESS_TOKEN = config.get_ssm_param(access_token_param)
+    return ACCESS_TOKEN
+
+
+def get_github_api_paged_data(url: str) -> list:
+    token = get_github_access_token()
+    page = 1
+    page_size = 100
+    page_items = 1
+    items = []
+    while page_items > 0:
+        page_items = 0
+        page_url = f"{url}?page={page}&per_page={page_size}"
+        headers = {"authorization": f"token {token}"}
+        response = requests.get(page_url, headers=headers).json()
+        if isinstance(response, list):
+            LOG.debug("Got item page %s for URL %s", page, url)
+            page_items = len(response)
+            for item in response:
+                items.append(item)
+        else:
+            LOG.error(response)
+        page += 1
+        time.sleep(4)
+
+    return items
+
+
+def get_github_org_members(org: str) -> list:
```

Comment:
While typehints are good for letting the programmer know what types are in use, it's even better to let the mypy compiler tell you what. 
If this file were run through mypy, it would tell you that `list` isn't the type, it's `typing.List`.

Reply:
I agree it's inconsistent, `str` is fine by mypy, but `list` and `dict` need to come from `typing`.
I expect it's a hangup from the legacy type schism when `str` changed from a function to a type, in the Dark Ages.

Reply:
Fixed in mypy branch.